test/entrypoint.sh

```diff
@@ -30,6 +30,14 @@ MYSQL_CONTAINER=1 $DIR/create_db.sh
 # Set up rabbitmq exchange
 rabbitmq-setup -server amqp://boulder-rabbitmq
 
+# Configure up SoftHSM
+wait_tcp_port boulder-hsm 5657
+PKCS11_PROXY_SOCKET="tcp://boulder-hsm:5657"
+pkcs11-tool --module=/usr/lib/libpkcs11-proxy.so --init-token --label token_label --so-pin 1234
```

Comment:
This should happen in build.sh, not in entrypoint.sh (which gets run for every command).